flink-connectors/flink-connector-filesystem/src/main/java/org/apache/flink/streaming/connectors/fs/StringWriter.java

```diff
@@ -40,12 +40,18 @@
 
 	private transient Charset charset;
 
+	private String rowDelimiter;
```

Comment:
Could you mark `rowDelimeter` and `rowDelimiterBytes` fields final (and set them in the constructor) as @StephanEwen suggested? `charsetName` could also be final.

Reply:
If rowDelimiterBytes was marked in final and set in the constructor,is it also necessary to move the the initialization of charset to the constructor ？Thanks ！

Reply:
Ok, I missed that. Thanks for the explanation